src/Sample.xoo

```diff
@@ -10,3 +10,4 @@ public class Sample1 {
     System.out.println("hello" + " xoo");
   }
 }
+// xoo
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:25771/organizations/default-organization/rules#rule_key=xoo%3AHasTag))

[See it in SonarQube](http://127.0.0.1:25771/project/issues?id=sample&branch=feature4&issues=AWFMQ5TSUW0BqEu_kI5Y&open=AWFMQ5TSUW0BqEu_kI5Y)

[ISSUE_KEY]: AWFMQ5TSUW0BqEu_kI5Y